Prolific/Prolific/Utils/User Engagement Manager/UserEngagementManager.m

```diff
@@ -8,6 +8,32 @@
 
 #import "UserEngagementManager.h"
 
+#import "UserBuilder.h"
+
 @implementation UserEngagementManager
 
++ (NSDecimalNumber *)computeKarmaValueForUserEngagementType:(UserEngagementType)userEngagementType {
```

Comment:
nice. looks good and no defaults

Reply:
For the future, you'd probably end up putting this logic on the server so Android / web doesn't have to do this. For the sake of the internship this makes sense of course